reports/Washington DC.md

```diff
@@ -35,7 +40,9 @@ id: dc-dc-11
 
 ### Australian news crew and protestors attacked by police | June 1st
 
-Protestors and an Australian reporter and cameraman were attacked by police while reporting on protests in DC.
+Protestors and an Australian reporter and cameraman were attacked by police while reporting on protests in DC. An officer charged directly at the journalist, and appeared to punch her. The journalist identified herself at media, and identified that tear gas and rubber bullets were used.
```

Comment:
Tiny little nitpick: but there shouldn't be a comma between "journalist" and "and" in the second sentence